doc/source/reference/spatial.rst

```diff
@@ -2,8 +2,3 @@
    :no-members:
    :no-inherited-members:
    :no-special-members:
-
-.. toctree::
-   :hidden:
-
-   spatial.distance
```

Comment:
This seems fine. `spatial.distance` and `spatial.transform` are clearly linked from `scipy.spatial`.